README.md

```diff
@@ -48,6 +48,12 @@ cd frontend
 npm run test
 ```
 
+xUnit is used for unit testing the backend code. To run all xUnit tests, run
+```
+cd backend
```

Comment:
Maybe the tests should be in a folder called "tests"? In that case it should be `cd backend/tests`

Reply:
They are, but you can run the command from the backend folder